lib/dal/src/func/authoring/save.rs

```diff
@@ -128,10 +128,15 @@ pub(crate) async fn save_func(
             }
         }
         FuncKind::Intrinsic | FuncKind::SchemaVariantDefinition | FuncKind::Unknown => {
-            return Err(FuncAuthoringError::NotWritable)
+            return Err(FuncAuthoringError::NotWritable(func.id))
         }
     }
 
+    WsEvent::func_saved(ctx, func.id)
+        .await?
+        .publish_on_commit(ctx)
+        .await?;
```

Comment:
With this in place, we should probably remove the return payload of `save_func` and `save_and_exec` func. After all, we want `layerdb` to handle persistence and the rebaser to finish for us to know when things are ready to be fetched.